### ISSUE TITLE
Sponsor div not move without clicking on button 

### ISSUE THREAD

**Issue by rovi555yadav** [axios/axios#5994]
### Is your feature request related to a problem? Please describe.

If Sponsor div moves without clicking on buttons and It moves by clicking mostly website moving features show looks nice

### Describe the solution you'd like

I want to change it into auto move and clicking on button is also to be there 

### Describe alternatives you've considered

![Axios - Google Chrome 11-10-2023 14_14_34](https://github.com/axios/axios/assets/92680028/876862d7-f4db-453b-9354-39dc9baf6751)

**Comment by theshrish46** [comment]
Hey are you working on this issue?

**Comment by Joseph2001-braganza** [comment]
I agree with @rovi555yadav  to this update. Mainteners look it into this fix.

**Comment by rovi555yadav** [comment]
Yes

**Comment by rivailruiz** [comment]
is there anyone working on it?

**Comment by DigitalBrainJS** [comment]
Autoscrolling is disabled in the carousel because the most generous (and/or newest) sponsors are located on the left in the most visible place. I thought about implementing automatic scrolling, which would be activated 15-20 seconds after the page loaded, but I didn’t get around to it.

**Comment by dlin2102** [comment]
Did you still need someone to implement automatic scrolling that's activated 15-20 seconds after the page has loaded?

**Comment by jasonsaayman** [comment]
closing this is solved
